rfps/rfp-fvm.md

```diff
@@ -1,24 +1,32 @@
 # FVM Tooling & Infrastructure RFP
 
-We are seeking proposals for various developer and infrastructure tooling to extend the functionality and usability of the [Filecoin Virtual Machine](https://github.com/filecoin-project/ref-fvm) (FVM).
-
 ## Overview
 
+We are seeking proposals for various developer and infrastructure tooling to extend the functionality and usability of the [Filecoin Virtual Machine](https://fvm.filecoin.io) (FVM).
+
 The Filecoin Virtual Machine (FVM) aims to combine smart contracts with co-location of storage and compute capabilities.
 
 The FVM is a WASM-based polyglot execution environment for IPLD data. It is designed to support native Filecoin actors written in languages that compile to WASM, as well as smart contracts written for foreign runtimes including the Ethereum Virtual Machine (EVM), Secure EcmaScript (SES), and eBPF.
 
-Refer to [FVM Website](https://fvm.filecoin.io/) for more details on the possibilities FVM will unlock, the current roadmap and links to the current [reference FVM and SDK](https://github.com/filecoin-project/ref-fvm) written in Rust.
+Refer to the resources section for more information on the FVM reference and specification, details on the possibilities FVM will unlock and the current roadmap.
+
+## Description
+
+The FVM is an integral and high-priority project for extending the functionality of Filecoin by bringing storage solutions to the blockchain and executing computation jobs or data pipelines on globally distributed data by pushing compute to the edges and coordinating its execution.
+
+Examples of future ideas include: async programming via continuations/futures, capability-orientation, time-locked transactions (using external beacons), parallel execution, native primitives for cross-chain interoperability, computation correctness proofs, and more.
 
 We want to build the best tech and tooling available to help enable future developers to build the use cases they envision on the FVM easily and seamlessly.
 
 We are therefore seeking proposals for various developer and infrastructure tooling to extend the functionality and usability of the FVM.
 
-These projects are critical to build the set of tools that we would need to provide in order to have an end-to-end builders process for people who are deploying smart contracts on FVM.
+These projects are essential to build the minimum set of tools that we would need to provide in order to have an end-to-end builders process for people who are deploying smart contracts on FVM.
+
+## Project Types
 
 The projects needed include (but are not limited to):
 
-- SDK's
+- SDKs
 
   - High-level Rust
```

Comment:
Can you add more detail around what high-level Rust means? (Unless it's something the target audience would immediately understand.)

Reply:
Yea - we already have a proposal for that one in, so it's kind of already a specific case

Reply:
Also I removed the comma because it kept messing up the formatting! shrug

Reply:
Fixed (both)!